src/global_mapper/global_mapper.h

```diff
@@ -26,25 +32,27 @@ class GlobalMapper {
   GlobalMapper(GlobalMapper&& rhs) = delete;
   GlobalMapper& operator=(GlobalMapper&& rhs) = delete;
 
-  const void IndToCoord(int ind, int ixyz[3]);
   void PushPointCloud(const PointCloud::ConstPtr& cloud_ptr);
   void Run();
 
   std::mutex cloud_mutex_;
   std::mutex map_mutex_;
+  std::mutex data_mutex_;
```

Comment:
Doesn't make much sense for mutexes, but in theory you could provide const accessors. Mainly it's just ugly/bad form though to provide direct access to public methods in non-POD structs (especially with the trailing underscore).